tests/test_repo.py

```diff
@@ -87,13 +88,15 @@ def test_make_command_executes(self, cli_runner, test_clean_filesystem):
         assert not error_messages
 
 
+@pytest.mark.skip
```

Comment:
These are the ghosts of a flawed pattern I once endorsed. I’ve since repented; see #639